src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -124,25 +124,35 @@ private Long getActionCode(String companyNumber, String requestId) {
     private ObjectionStatus getObjectionStatusForCreate(Long actionCode, String companyNumber, String logContext) {
         ObjectionStatus objectionStatus = ObjectionStatus.OPEN;
         try {
-            actionCodeValidator.validate(actionCode, logContext);
+            validateActionCode(actionCode, companyNumber, logContext);
```

Comment:
We probably don't need this method anymore we could call validateActionCode from the createObjection method as we are now doing from isEligible.

Reply:
it was split out to improve readability of the createObjection method. If we merge them together it will make the createObjection longer and less readable.